demo/src/Components/Tooltip/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+
+import Tooltips from './Tooltips'
+
+class TooltipsContainer extends Component {
+  constructor() {
+    super()
+    this.state = {
+      openTooltip: {
+        north: false,
+        south: false,
+      },
+    }
+  }
+
+  handleClick = (direction) => {
+    const newState = Object.keys(this.state.openTooltip).reduce((acc, curr) => {
+      acc[curr] = false
+      return acc
+    }, {})
+    this.setState({
```

Comment:
For the demo just put two directions as static props on two examples.  No need to mess around with state or anything here.

Reply:
done